react-main/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,15 +1,47 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 
 import Square from './components/Square';
 
 class Board extends Component {
-  renderSquare = i => <Square />;
+  state = {
+    squares: Array(9).fill(null),
+    isNext: true
+  };
+
+  calculateWinner = squares => {
```

Comment:
This method can be extracted to a self-contained function. You can also move that function to a file named utils.js (we place functions there) and import it here.